contrib/deploy_timesketch.sh

```diff
@@ -79,6 +79,7 @@ curl -s $GITHUB_BASE_URL/docker/release/config.env > timesketch/config.env
 # Fetch default Timesketch config files
 curl -s $GITHUB_BASE_URL/data/timesketch.conf > timesketch/etc/timesketch/timesketch.conf
 curl -s $GITHUB_BASE_URL/data/tags.yaml > timesketch/etc/timesketch/tags.yaml
+curl -s $GITHUB_BASE_URL/data/plaso.mappings > timesketch/etc/timesketch/plaso.mappings
```

Comment:
This file should be available in plaso-data check if you need to set data path correctly

Reply:
this is another mapping, since we moved all TS specifics over to Timesketch. The plaso mapping file is used in the Timesketch output module, and the default ES module, but we are now generating the index in TS so that we can more easily control the mapping file there, since this is more of a TS issue than a plaso one, so yes this is a separate file

Reply:
@berggren I'll follow up with @kiddinn off thread, question is resolved for this PR